lib/core/util/asset_constants.dart

```diff
@@ -1,3 +1,35 @@
 abstract class AssetConstants {
+  static const String environmentVariables = '.env';
   static const String splashScreen = 'assets/splash_screen.json';
+  static const String companyLogo = 'assets/images/company-logo.svg';
+  static const String sliverBackground = 'assets/images/background.webp';
+  static const String searchIcon = 'assets/images/recipe_search_icon.png';
+  static const String montserratFont = 'Montserrat';
+  static const String timerIcon = 'assets/images/clock_icon.webp';
+  static const String instagramIcon = 'assets/social/instagram.svg';
+  static const String facebookIcon = 'assets/social/facebook.svg';
+  static const String twitterIcon = 'assets/social/twitter.svg';
+
+  static const Map<String, String> recipeFlags = {
+    'cheap': 'assets/images/flags/cheap.svg',
+    'dairy_free': 'assets/images/flags/dairy-free.svg',
+    'gluten_free': 'assets/images/flags/gluten-free.svg',
+    'healthy': 'assets/images/flags/healthy.svg',
+    'low_foodmap': 'assets/images/flags/lowfm.svg',
+    'popular': 'assets/images/flags/popular.svg',
+    'sustainable': 'assets/images/flags/sustainable.svg',
+    'vegan': 'assets/images/flags/vegan.svg',
+    'vegetarian': 'assets/images/flags/vegetarian.svg',
+  };
+  static const Map<String, String> recipeFlagsTooltip = {
```

Comment:
If you choose the enum option, you can add the tooltip as a new parameter to the constructor like
`const RecipeFlags(this.asset, this.tooltip)`

Reply:
Done